packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1008,6 +1008,15 @@ export const components = ({
               },
             },
 
+            [`&.${chipClasses.colorInfo}`]: {
+              backgroundColor: odysseyTokens.HueBlue50,
+              color: odysseyTokens.HueBlue600,
+
+              "&::before": {
+                backgroundColor: odysseyTokens.HueBlue600,
+              },
+            },
```

Comment:
We don't have these colors as first-class citizens in Design Tokens?

Reply:
@KevinGhadyani-Okta I think there was a recent discussion with Alex about this but not sure that we landed on the exact approach yet. IMO, something we should tokenize when we have clarity.

Reply:
I can change the background to `PalettePrimaryLighter` but, the font color doesn't have a 'palette' color associated